### PR TITLE
Rebuild /ai/services page

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Rebuild `/ai/services` page
- Update learn more section to include dynamic AI blog post list

## QA

- View page at: https://ubuntu-com-9219.demos.haus/ai/services
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://docs.google.com/document/d/19Mx1zRJKqH2hz5kRF_oDsgyFUw67FgJvW4I_OGda-wk/edit#)

## Issue / Card

Fixes [#3767](https://github.com/canonical-web-and-design/web-squad/issues/3767)

### CODE REVIEW

**Comment by spencerbygraves** [comment]
Thanks @bethcollins92, a couple of changes from me please.

Can the "Learn more..." title in the last section be h2 please. Can we also update the partners section, I've moved the logos and balanced out the sizes a bit.

Thanks.

image

**Comment by spencerbygraves** [comment]
@bethcollins92 I didn't mean to remove the button! Thanks.

**Comment by bethcollins92** [comment]
Thanks @spencerbygraves 👍🏼  ready for a re-review please

**Comment by spencerbygraves** [comment]
Thanks @bethcollins92 👍

File: templates/ai/services.html [hunk 1]
```diff
@@ -7,316 +7,306 @@
...
+      <p class="p-heading--4">4&ndash;days to success with machine learning on Kubeflow</p>
+      <p>Grow end-to-end machine learning knowledge within your team, with enterprise training by Canonical.</p>
+      <p>Align your sysadmins, devops, data engineers and data scientists with the latest MLOps solutions.</p>
+      <p>Any infra and level of expertise. Tailored to your use cases. Up to 15 attendees. 4-day on-premise training for $29.500.</p>
       <p>
-        Prototype new solutions and identify a candidate solution that seems to be the best fit for the use case under consideration. Data engineers will prepare the data, data scientists will discuss and select appropriate features and machine learning algorithms, and machine Learning engineers will design, build and perform preliminary tests on your prototype neural network. Time permitting, the iterative process of design and discovery on the data and the neural network model will start.
+        <a class="p-link--external p-button--neutral" href="https://assets.ubuntu.com/v1/ee5a96f3-Kubeflow+Training+Curriculum.pdf?_ga=2.202291033.951646160.1607941648-154096706.1586788067">See Curriculum</a>
```
Comment (pmahnke) [inline templates/ai/services.html:122]:
```suggestion
        <a class="p-button--neutral" href="https://assets.ubuntu.com/v1/ee5a96f3-Kubeflow+Training+Curriculum.pdf?_ga=2.202291033.951646160.1607941648-154096706.1586788067">See the curriculum</a>
```

We don't do external links for assets.u.c

File: templates/ai/services.html [hunk 2]
```diff
@@ -7,316 +7,306 @@
...
-      </p>
+<section class="p-strip--square-darksuru is-dark">
+  <div class="row u-equal-height">
+    <div class="col-7 u-vertically-center">
+      <div>
+        <h2>Easy Kubeflow operations</h2>
+        <p>Charmed Kubeflow integrates the 30+ apps that make up Kubeflow with ops code to provide the best Kubeflow experience, from deployment to day-2 operations. </p>
+        <br />
+          <a class="p-link--external p-button--positive" href="https://charmed-kubeflow.io">Visit Charmed-Kubeflow.io</a>
+          <a class="p-link--external p-link--inverted" href="https://assets.ubuntu.com/v1/197d9867-Charmed_Kubeflow_Kubernetes_10.11.20+%282%29.pdf">Download datasheet</a>
```
Comment (pmahnke) [inline templates/ai/services.html:137]:
```suggestion
          <a class="p-link--inverted" href="https://assets.ubuntu.com/v1/197d9867-Charmed_Kubeflow_Kubernetes_10.11.20+%282%29.pdf">Download the datasheet</a>
```

again, not external, and we need definate articles

File: templates/ai/services.html [hunk 3]
```diff
@@ -7,316 +7,306 @@
...
-      </p>
+  <div class="row u-vertically-center">
+    <div class="col-7">
+      <h1>Enterprise MLOps solutions</h1>
+      <p class="p-heading--4">Consulting, support, training and managed services for AI/ML at scale</p>
+      <p>Accelerate your artificial intelligence strategy with data science + infra + ops expertise, delivered by Canonical and trusted partners.</p>
       <p>
-        <a class="p-button--positive js-invoke-modal" href="/kubeflow/contact-us?product=ai-index">Speak to the experts</a>
-        <a class="p-link--external" href="/engage/starting-with-ai?utm_source=bubble&utm_campaign=CY19_DC_AI_Whitepaper_AIConsulting">Download the Getting started with AI whitepaper</a>
+        <a class="p-button--positive js-invoke-modal" href="/ai/contact-us?product=ai-index">Talk to experts</a>
```
Comment (pmahnke) [inline templates/ai/services.html:16]:
```suggestion
        <a class="p-button--positive js-invoke-modal" href="/ai/contact-us?product=ai-index">Talk to the experts</a>
```

add 'the'

**Review by pmahnke — CHANGES_REQUESTED** [review]
Some changes inline.  I also asked about the double list of blog posts @RFMVasconcelos and @bethcollins92

**Comment by bethcollins92** [comment]
@pmahnke Added the Kubeflow news section, if you would mind taking a look and QA-ing? The thank you message appears now, but I'm not too sure if/how to check the form is submitting correctly. I copied the marketo form ID from the kubeflow-news.com page

**Comment by pmahnke** [comment]
@bethcollins92 the form doesn't work, you are missing a lot of hidden fields

```
      <input type="hidden" name="Consent_to_Processing__c" value="Yes">
      <input type="hidden" name="formid" value="3415">
      <input type="hidden" name="formVid" value="3415">
      <input type="hidden" name="lpId" value="">
      <input type="hidden" name="munchkinId" value="066-EOV-335">
      <input type="hidden" name="lpurl" value="">
 ```

**Comment by bethcollins92** [comment]
@pmahnke thanks, added those in now

**Review by pmahnke — APPROVED** [review]
works! thanks

**Pull request merged**